webgpu.h

```diff
@@ -507,6 +507,23 @@ typedef enum WGPUErrorType {
     WGPUErrorType_Force32 = 0x7FFFFFFF
 } WGPUErrorType WGPU_ENUM_ATTRIBUTE;
 
+/**
+ * See @ref WGPURequestAdapterOptions::featureLevel.
+ */
+typedef enum WGPUFeatureLevel {
+    /**
+     * `0x00000001`.
+     * "Compatibility" profile which can be supported on OpenGL ES 3.1.
```

Comment:
nit: and D3D11

Reply:
I'm unclear on this, I thought we came to the conclusion D3D11 could actually support core? Or did that turn out not to be true?

Reply:
We guessed that maybe it is possible but we never tried. So for now it is safe to assume that D3D11 only provides compat.